common/nodejs-utils/src/package_json.rs

```diff
@@ -6,24 +6,37 @@ use std::io::BufReader;
 use std::path::Path;
 use thiserror::Error;
 
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Debug, Default, Clone)]
 pub struct PackageJson {
     pub name: String,
     pub version: Option<Version>,
     pub engines: Option<Engines>,
+    pub scripts: Option<Scripts>,
     pub main: Option<String>,
     pub dependencies: Option<HashMap<String, String>>,
     #[serde(rename = "devDependencies")]
     pub dev_dependencies: Option<HashMap<String, String>>,
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Debug, Default, Clone)]
 pub struct Engines {
     pub node: Option<Requirement>,
     pub yarn: Option<Requirement>,
     pub npm: Option<Requirement>,
 }
 
+#[derive(Deserialize, Debug, Default, Clone)]
```

Comment:
I actually had that in a previous commit, but I took it out, since other package.json fields tend to favor `camelCase`. Since the casing isn't consistent, I think I'd rather do this case-by-case.